packages/common/src/utils/sanitizeHtml.ts

```diff
@@ -1,5 +1,47 @@
 import sanitize from 'sanitize-html';
 
+/**
+ * A list of tags for which style attributes are allowed, but only
+ * for attributes commonly useful for text styling (e.g centering text,
+ * changing colors, etc).
+ */
+const tagNamesAllowingTextStyling = [
+    'span', // Also required for comment @mentions to be styled appropriately
+    'a',
+    'p',
+    'b',
+    'strong',
+    'em',
+    'i',
+    'td',
+    'code',
+    'h1',
+    'h2',
+    'h3',
+    'h4',
+    'h5',
+    'h6',
+];
+
+/**
+ * Defines a list of CSS properties and value RegExps, which will be allowed as part of the
+ * style attribute in the above tags.
+ */
+const allowedTextStylingProperties: NonNullable<
+    sanitize.IOptions['allowedStyles']
+>[string] = {
+    'font-size': [/^\d+(?:px|em|rem|%)$/], // 12px, 1em, 20%
```

Comment:
There's probably a good case for just making these all `/.*/`, but basic validation is easy enough regardless (with a couple of exceptions at the end)